master/getting-started/kubernetes/tutorials/using-calicoctl.md

```diff
@@ -2,53 +2,13 @@
 title: Using calicoctl in Kubernetes
```

Comment:
This entire guide sorta feels useless now - it doesn't tell me how to use calicoctl, it just links me to how to install it.

Are we planning on adding more content in the future?

Reply:
Well, actually, it only told people how to install it _before_ (despite the title) and now there is only a small amount of content left which seemed somewhat useful, so I didn't want to delete this page. It could be also that later on, we enhance this page to talk more about the use of calicoctl in K8s. If you want we can make a story for that.

Reply:
I've added a story to track this issue.